django_socio_grpc/tests/fakeapp/serializers.py

```diff
@@ -239,3 +242,20 @@ class Meta:
 
 class NoMetaSerializer(proto_serializers.ProtoSerializer):
     my_field = serializers.CharField()
+
+
+class EnumServiceSerializer(proto_serializers.ModelProtoSerializer):
+    class Meta:
+        model = EnumModel
+        proto_class = fakeapp_pb2.EnumServiceResponse
+        fields = "__all__"
+
+
+class EnumServiceAnnotatedSerializerSerializer(proto_serializers.ProtoSerializer):
+    char_choices: Annotated[serializers.ChoiceField, EnumModel.MyTestStrEnum] = (
```

Comment:
to test if working on a specific field use a new name like char_choices_in_serializer so it's assure it works without model attribute existing.